lib/github-glue.ts

```diff
@@ -314,7 +314,7 @@ export class GitHubGlue {
             repo: "git",
         });
         const result: IPRCommit[] = [];
-        response.data.map((cm: octokit.PullsListCommitsResponseItem) => {
+        response.data.map((cm) => {
```

Comment:
This seems to be another ESLint-related change, rather than an Octokit-related one?

Reply:
See the previous.